sources/AppBundle/Event/Model/Event.php

```diff
@@ -712,6 +717,24 @@ public function setWaitingListUrl($waitingListUrl)
         return $this;
     }
 
+    public function hasPricesDefinedWithVat()
+    {
+        return (bool) $this->hasPricesDefinedWithVat;
+    }
```

Comment:
On est en PHP 7 maintenant donc tu peux faire ça plutôt :
```suggestion
    public function hasPricesDefinedWithVat(): bool
    {
        return $this->hasPricesDefinedWithVat;
    }
```